playground/AspireEventHub/EventHubs.AppHost/ehstorage.module.bicep

```diff
@@ -0,0 +1,67 @@
+targetScope = 'resourceGroup'
+
+@description('')
+param location string = resourceGroup().location
+
+@description('')
+param principalId string
+
+@description('')
+param principalType string
+
+
+resource storageAccount_X1L2R0Ykm 'Microsoft.Storage/storageAccounts@2022-09-01' = {
```

Comment:
It's not an issue but that's a weird variable name. Or do these resources need to be unique for a whole deployment?

Reply:
I believe they are generated by the Azure CDK.